content/en/platform/corda/4.8/enterprise/node-docker-deployments.md

```diff
@@ -0,0 +1,188 @@
+        -v /home/user/docker/certificates:/opt/corda/certificates \
+        corda/corda-zulu-java1.8-4.8:latest config-generator --generic --exit-on-generate
+```
+
+Several environment variables must also be passed to the container to allow it to register:
+
+
+* `MY_LEGAL_NAME` - The X500 to use when generating the config. This must be the same as registered with the Zone.
+* `MY_PUBLIC_ADDRESS` - The public address to advertise the node on.
+* `NETWORKMAP_URL` - The address of the Zone’s network map service (this should be provided to you by the Zone).
+* `DOORMAN_URL` - The address of the Zone’s doorman service (this should be provided to you by the Zone).
+* `NETWORK_TRUST_PASSWORD` - The password to the Zone Trust Root (this should be provided to you by the Zone).
+* `MY_EMAIL_ADDRESS` - The email address to use when generating the config. This must be the same as registered with the Zone.
+* `SSHPORT` - The port to use for SSH.
+* `RPC_USER` - The RPC username.
+
+There are some optional variables which allow customisation of the generated config:
+
+
+* `MY_P2P_PORT` - The port to advertise the node on (defaults to 10200). If changed, ensure the container is launched with the correct published ports.
+* `MY_RPC_PORT` - The port to open for RPC connections to the node (defaults to 10201). If changed, ensure the container is launched with the correct published ports.
+
+Once the container has finished performing the initial registration, the node can be started as normal
+
+```shell
+docker run -ti \
+        --memory=2048m \
+        --cpus=2 \
+        -v /home/user/docker/config:/etc/corda \
+        -v /home/user/docker/certificates:/opt/corda/certificates \
+        -v /home/user/docker/persistence:/opt/corda/persistence \
+        -v /home/user/docker/logs:/opt/corda/logs \
+        -v /home/user/corda/samples/bank-of-corda-demo/build/nodes/BankOfCorda/cordapps:/opt/corda/cordapps \
+        -p 10200:10200 \
+        -p 10201:10201 \
+        corda/corda-zulu-java1.8-4.8:latest
+```
+
+# Performing Database Migrations
+
+The image contains the database-migration tool. It is possible to run this in two modes within a container.
+
+## Generating Migration Jars
+
+In this mode, the database-migration tool will scan the provided CorDapps, and generate corresponding migration jars. These jars will be placed alongside
+the source CorDapps. In this example, there are two CorDapps provided `corda-insurance.jar` and `corda-kyc.jar`
+
+```shell
+docker run -ti \
+        -v /home/user/docker/docker/config:/etc/corda \
+        -v /home/user/docker/docker/certificates:/opt/corda/certificates \
+        -v /home/user/docker/docker/persistence:/opt/corda/persistence \
+        -v /home/user/docker/docker/logs:/opt/corda/logs \
+        -v /home/user/corda/samples/bank-of-corda-demo/build/nodes/BankOfCorda/cordapps:/opt/corda/cordapps \
+        entdocker.software.r3.com/corda-enterprise-java1.8-4.8:latest db-migrate-create-jars
```

Comment:
The  image entdocker.software.r3.com/corda-enterprise-java1.8-4.8:latest no longer seems to exist.
However, corda/corda-enterprise-alpine-zulu-java1.8-4.8.6:latest, from docker hub does.